README.md

```diff
@@ -13,6 +13,8 @@ Crystal is a programming language with the following goals:
 ## Build
 
 ```
+$ git clone git@github.com:crystal-lang/crystal-book.git
```

Comment:
Not everybody has their SSH key setup with Github, let's use the HTTPS URL here.